src/libs/Navigation/getTopmostReportId.js

```diff
@@ -10,6 +10,9 @@ import lodashGet from 'lodash/get';
  * @returns {String | undefined} - It's possible that there is no report screen
  */
 function getTopmostReportId(state) {
+    if (!state) {
```

Comment:
@parasharrajat we found a case where state could arrive undefined . This is to avoid issues in the future where this functions is called and we don't know the specific value of state that could arrive.